src/olympia/files/utils.py

```diff
@@ -805,6 +806,45 @@ def update_version_number(file_obj, new_version_number):
     shutil.move(updated, file_obj.file_path)
 
 
+def write_crx_file(chunks, storage, loc):
```

Comment:
Maybe call it `convert_crf_to_xpi`? Also, I'd use `target` instead of `loc` or at least `location` which makes reasoning about what the heck this is easier.